src/com/projectkorra/projectkorra/board/BendingBoardManager.java

```diff
@@ -45,15 +50,27 @@ public static void reload() {
 	}
 
 	private static void initialize(boolean initial) {
-		if (initial) loadDisabledPlayers();
+		if (initial) {
+			loadDisabledPlayers();
+		}
+		
 		enabled = ConfigManager.getConfig().getBoolean("Properties.BendingBoard");
 		disabledWorlds.clear();
 		disabledWorlds.addAll(ConfigManager.getConfig().getStringList("Properties.DisabledWorlds"));
 	}
+	
+	/**
+	 * Check if the player has the BendingBoard toggled off (disabled)
+	 * @param player {@link Player} to check toggled
+	 * @return true if player has the BendingBoard toggled off
+	 */
```

Comment:
If you're going to add documentation to this function could you pls pls add it to the other ones as well while you're at it :D

Reply:
I added documentation because I made that function lol I haven't looked at what the other functions actually do so I wouldn't want to write wrong javadocs